tardis/io/atom_data/base.py

```diff
@@ -164,7 +164,7 @@ def from_hdf(cls, fname=None):
                 try:
                     dataframes[name] = store[name]
                 except KeyError:
-                    logger.debug("Dataframe does not contain NAME column")
+                    logger.debug(f"Dataframe does not contain {name} column")
```

Comment:
@andrewfullard @Rodot- This change needs to be merged. What should be done here? The PR will remain Draft.